### PR TITLE
Add tvl for crosscurve

### PR DESCRIPTION
**Pull request by spiehdid**
##### List of audit links if any:
https://smartstate.tech/clients/eywa.html
https://hexens.io/audits#eywa

##### Logo (High resolution, will be shown with rounded borders):
logo-symbol

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/crosscurve exports TVL: 
        
 
 
    fantom                    1.28 M
    base                      281.86 k
    bsc                       202.71 k
    avax                      170.62 k
    xdai                      167.31 k
    arbitrum                  159.79 k
    polygon                   132.14 k
    ethereum                  120.78 k
    optimism                  77.22 k
    
    total                    2.60 M

**Comment by llamatester** [comment]
The adapter at projects/crosscurve exports TVL: 
        
 
 
    fantom                    1.28 M
    base                      287.93 k
    bsc                       203.23 k
    avax                      170.87 k
    xdai                      167.31 k
    arbitrum                  160.02 k
    polygon                   136.17 k
    ethereum                  120.81 k
    optimism                  76.93 k
    
    total                    2.61 M

File: projects/crosscurve/index.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  xdai: {
+    tvl: sumTokensExport({
+      owner: "0xac8f44ceca92b2a4b30360e5bd3043850a0ffcbe",
+      tokens: [
+        "0x1337bedc9d22ecbe766df105c9623922a27963ec", // x3CRV
+      ],
+    }),
+  },
+  fantom: {
+    tvl: fantom,
```
Comment (waynebruce0x) [inline projects/crosscurve/index.js:129]: this should be removed because this is double counting the tvl on other chains

**Comment by llamatester** [comment]
The adapter at projects/crosscurve exports TVL: 
        
 
 
    base                      290.05 k
    bsc                       188.29 k
    avax                      170.27 k
    xdai                      169.72 k
    arbitrum                  155.81 k
    polygon                   134.72 k
    ethereum                  120.78 k
    optimism                  74.93 k
    
    total                    1.30 M

File: projects/crosscurve/index.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
+const { sumTokensExport } = require("../helper/unwrapLPs");
+
+module.exports = {
+  ethereum: {
+    tvl: sumTokensExport({
+      owner: "0xac8f44ceca92b2a4b30360e5bd3043850a0ffcbe",
+      tokens: [
```
Comment (g1nt0ki) [inline projects/crosscurve/index.js:7]: hi @spiehdid thanks for the PR, is there some way to pull the token list on chain? like from this whitelist contract? https://etherscan.io/address/0x5cF21A9899e679448284427C0Baca807fAd6bB8F#code

I tried to use `tokens` but for some reason, I keep getting error

Comment (spiehdid) [reply projects/crosscurve/index.js:7]: There are a lot of extra tokens

Comment (g1nt0ki) [reply projects/crosscurve/index.js:7]: sorry, I dont understand. you mean, there are more tokens than those that are part of the whitelist?

Comment (spiehdid) [reply projects/crosscurve/index.js:7]: There are many tokens in the white list that should not be included in the TVL

**Pull request merged**